lib/widgets/atoms/reminder_banner.dart

```diff
@@ -0,0 +1,125 @@
+import 'package:flutter/material.dart';
+import 'package:mm_flutter_app/constants/app_constants.dart';
+
+class ReminderBanner extends StatelessWidget {
+  final String titleText;
+  final String subtitleText;
+  final String ctaText;
+
+  const ReminderBanner({
+    Key? key,
+    required this.titleText,
+    required this.subtitleText,
+    required this.ctaText
+  }) : super(key: key);
+
+  Widget _buildTextColumn(context){
+    return Expanded(
+      flex: 1,
+      child: Column(
+          crossAxisAlignment:  CrossAxisAlignment.start,
+          mainAxisSize: MainAxisSize.min,
+          children: <Widget> [
+              Text(
+                titleText,
+                style: TextStyles.cardTitle(context),
+                textAlign: TextAlign.start,
+                softWrap: true
+              ),
+              const SizedBox(height: Insets.widgetSmallestInset),
+              Text(
+                subtitleText,
+                style: TextStyles.cardSubtitle(context),
+                textAlign: TextAlign.start,
+                softWrap: true
+              ),
+          ],
+        )
+      );
+  }
+
+  Widget _buildImageAndButtonColumn(context){
+    return Expanded(
+      flex: 1,
+      child: Column(
+          mainAxisSize: MainAxisSize.min,
+          children: <Widget> [
+            const SizedBox(
+                child : Image(
+                image: AssetImage("assets/images/StockImage1.png"),
+                height: Dimensions.reminderBannerImageHeight,
+            )),
+            InkWell(
+            onTap:  () => {
+              debugPrint("hi")
+            },
+            child: Row(children: [
+                const Icon(Icons.arrow_forward),
+                Text( ctaText, 
+                style: TextStyles.cardTitle(context)),
```

Comment:
I like putting a comma at the end of each parameter which also allows the formatter to do it's job. Check out the official docs for their take on this https://docs.flutter.dev/tools/formatting#using-trailing-commas

Reply:
good point, thanks for the pointer